Packs/EDL/ReleaseNotes/3_1_21.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Block Domain - External Dynamic List
+
+EDL task supports appending indicators' tags.
```

Comment:
Elaborate more about the change (appending vs replacing)
comment relevant for all the release notes